### PR TITLE
Update convert-user-mailbox-to-shared-mailbox.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line :The Rules are intact after the mailbox is converted as a shared mailbox.:

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ When you convert a user's mailbox to a shared mailbox, all of the existing email
 
 - Don't delete the old user's account. That's required to anchor the shared mailbox. If you've already deleted the user account, see [Convert the mailbox of a deleted user](#convert-the-mailbox-of-a-deleted-user).
 
+- The Rules are intact after the mailbox is converted as a shared mailbox.
```
Comment (JohanFreelancer9) [inline Office365-Admin/email/convert-user-mailbox-to-shared-mailbox.md]:
```suggestion
- The rules are intact after the mailbox is converted to a shared mailbox.
```

**Comment by mypil** [comment]
@msbemba - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@cmcatee-MSFT - PR has been copy edited and is ready for final review. This is based on issue #630 

Thank you.

**Pull request merged**